Automattic/simplenote-android
# Add Style Setting (Part 1)

### Fix
This is the first part in a series of pull requests to add a style setting to the app.  The pull requests are split into parts in an attempt to make the changes smaller.  This first part will be larger than the subsequent parts since it's setting the foundation for adding other styles.  Also, there are parts of the code that may seem unnecessary (e.g. `switch` statements with one or two cases).  Those are structured so with the additional styles in mind.

This part adds the **_Default_** and **_Classic_** styles.  There are subtle differences between the two styles.  The most noticeable ones are the shade of blue and the **_Classic_** style tints the action icons blue.  See the screenshots below for illustration.

![add_style_setting_all_01](https://user-images.githubusercontent.com/3827611/89299994-c4f86000-d624-11ea-96a6-6ae18cc1a12b.png)

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item in drawer.
3. Tap ***Style*** item under ***Appearance*** section.
4. Notice ***Style*** screen with ***Default*** and ***Classic*** styles.
5. Notice ***Default*** style is selected and app has ***Default*** styling.
6. Tap ***Classic*** style.
7. Notice ***Classic*** style is selected and app has ***Classic*** styling.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
